modules/python/src2/hdr_parser.py

```diff
@@ -260,7 +268,7 @@ def parse_class_decl(self, decl_str):
             modlist.append("=" + macro_arg)
             l = l[:npos] + l[npos3+1:]
 
-        l = self.batch_replace(l, [("CV_EXPORTS_W", ""), ("CV_EXPORTS", ""), ("public virtual ", " "), ("public ", " "), ("::", ".")]).strip()
+        l = self.batch_replace(l, [("CV_EXPORTS_W", ""), ("CV_EXPORTS", ""), ("GAPI_EXPORTS_W", ""), ("GAPI_EXPORTS", ""), ("public virtual ", " "), ("public ", " "), ("::", ".")]).strip()
```

Comment:
To keep code maintainable/mergable it is better to put G-API specifics on a separate single line (because they are just aliases):
```
# G-API specific aliases
l = self.batch_replace(l, [
        ("GAPI_EXPORTS", "CV_EXPORTS"),
        ("GAPI_EXPORTS_W", "CV_EXPORTS_W"),
        ...
        ('defined(GAPI_STANDALONE)', '0'),
    ])

... existed unchanged OpenCV code ...
```

Reply:
> args should be compile_args in .apply() methods.

Yep but we have a function with the same name. :)

Reply:
@TolyaTalamanov , please consider https://github.com/opencv/opencv/pull/17493/files#r447131187 (it will make it easier to backport changes between master and 3.4.

Reply:
Done